src/pe_reports/metrics.py

```diff
@@ -12,25 +22,571 @@ def __init__(self, start_date, end_date, org_uid):
         self.start_date = start_date
         self.end_date = end_date
         self.org_uid = org_uid
-        self.query_cyberSix_creds = query_cyberSix_creds(org_uid, start_date, end_date)
+        c6_df = query_cyberSix_creds(org_uid, start_date, end_date)
+        c6_df.loc[
+            c6_df["breach_name"] == "", "breach_name"
+        ] = "Cyber_six_" + pd.to_datetime(c6_df["breach_date"]).dt.strftime("%m/%d/%Y")
+        c6_df["description"] = (
+            c6_df["description"].str.split("Query to find the related").str[0]
+        )
+        c6_df["password_included"] = np.where(c6_df["password"] != "", True, False)
+        self.query_cyberSix_creds = c6_df
```

Comment:
Does it make any sense to move the cyberSix code above into `query_cyberSix_creds()` or is it specific to this metrics code?

Reply:
Moved code to db_query.py as suggested

Reply:
@DJensen94 - Just a reminder that our convention is to allow the person who starts a conversation be the one to resolve it.